prometheus/README.md

```diff
@@ -14,7 +14,9 @@ You can use this initialization action to create a new Dataproc cluster with Pro
     CLUSTER_NAME=<cluster_name>
     gcloud dataproc clusters create ${CLUSTER_NAME} \
         --region ${REGION} \
-        --initialization-actions gs://goog-dataproc-initialization-actions-${REGION}/prometheus/prometheus.sh
+        --num-masters 3 \
+        --metadata "run-on-master=true" \
+        --initialization-actions gs://goog-dataproc-initialization-actions-${REGION}/kafka/kafka.sh,gs://goog-dataproc-initialization-actions-${REGION}/prometheus/prometheus.sh
```

Comment:
I think we need example of 2 commands, w/ and w/o Kafka init action. If Kafka init action is not installed, then we should set metadata value to disable Prometheus integration w/ Kafka.

Reply:
Can we disable Prometheus integration with Kafka? From the init script it seems like Kafka is always used. Doesn't seem like there is a flag to disable it. Do you mean we should introduce a new metadata?